docs/rules/no-unused-services.md

```diff
@@ -0,0 +1,40 @@
+# no-unused-services
+
+Disallow unused service injections.
+
+By removing unused service injections, we can reduce the amount of code we have and improve code readability.
+
+**Note**: This rule cannot detect service usage in the corresponding handlebars template file for a JavaScript class, which can thus cause the rule to exhibit false positives.
```

Comment:
I realized there are some more situations that can exhibit false positives so let's list them out here:

```suggestion
**Note**: This rule can exhibit false positives when an injected service is only used in:

* The corresponding handlebars template file for a controller or component
* A mixin or other class that the current class extends from
```